bucket/visualvm.json

```diff
@@ -0,0 +1,41 @@
+{
+    "version": "2.0.2",
+    "description": "Java troubleshooting tool",
+    "homepage": "https://visualvm.github.io/index.html",
+    "license": "GPL-2.0-only",
+    "suggest": {
+        "JDK": [
+            "java/oraclejdk",
+            "java/openjdk"
+        ]
+    },
+    "url": "https://github.com/oracle/visualvm/releases/download/2.0.2/visualvm_202.zip",
+    "hash": "e189e847cf299d0e5336690428b95ad1b9f3b88d4502bfc67e316502b56af56f",
+    "extract_dir": "visualvm_202",
+    "pre_install": [
+        "$conf = Get-Content \"$dir\\etc\\visualvm.conf\"",
+        "$conf = $conf -replace '^(visualvm_default_userdir=).*$', \"`$1\"\"$dir\\user\"\"\"",
+        "$conf = $conf -replace '^(visualvm_default_cachedir=).*$', \"`$1\"\"$dir\\cache\"\"\"",
```

Comment:
YOu cannot do this all the time.

What if user change it?? You will replace his desired configuration.

Always encapsure this under $persist_dir check